eqc/multisig_eqc.erl

```diff
@@ -4,12 +4,12 @@
 
 -export([prop_multipubkey_test/0]).
 
--define(HASHTYPE, sha256).
+-define(NETWORK, mainnet).
```

Comment:
Would be good to have a generator for network types. `testnet` is actually more interesting to test, since `mainnet` is the default in many places and would be the one created if network type is not properly propagated somewhere.